### PR TITLE
feat: adds glossary 

### PR DESCRIPTION
**Pull request by Lymah123**
## Description
- This is the submission to issue #200 

## What type of PR is this? (check all applicable)

- [x] 🍕 Feature
- [ ] 🐛 Bug Fix
- [x] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Added tests?

- [ ] 👍 yes
- [ ] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md
- [x] 📓 docs.opensauced.pizza
- [ ] 🍕 dev.to/opensauced
- [ ] 📕 storybook
- [ ] 🙅 no documentation needed

### CODE REVIEW

**Comment by Lymah123** [comment]
Hi @BekahHW and @CBID2 , I fixed the conflicting files with this PR.

**Comment by Lymah123** [comment]
Hi, @BekahHW and @CBID2, you can now review. I will be waiting!

Thank you.

**Comment by CBID2** [comment]
> Hi @BekahHW and @CBID2 , I fixed the conflicting files with this PR.

Hi @Lymah123. Can you add the issue number to the `Related Documents and Tickets" section instead of the Description?

**Review by CBID2 — APPROVED** [review]
Looks great @Lymah123! :)

**Comment by Lymah123** [comment]
> > Hi @BekahHW and @CBID2 , I fixed the conflicting files with this PR.
> 
> Hi @Lymah123. Can you add the issue number to the `Related Documents and Tickets" section instead of the Description? It'll make it easier for the PR and issue to close.

Hi @CBID2,  how do you mean? I didn't get that.

**Comment by adiati98** [comment]
@Lymah123 you added a [setting-up-a-new-repository.md](docs/maintainers/setting-up-a-new-repository.md) here. I don't think that it is part of the newest Maintainer section.

@BekahHW can you confirm this?

File: docs/contributing/technical/resolve-merge-conflicts.md [hunk 1]
```diff
@@ -11,6 +11,11 @@ Pretty often when opening a pull request it is very likely to run into merge con
 To better illustrate the commands listed here at will use commits and screenshots from [open-sauced#1078](https://github.com/open-sauced/open-sauced/pull/1078).
 
 
+## Repository setup
```
Comment (adiati98) [inline docs/contributing/technical/resolve-merge-conflicts.md:14]: @Lymah123 There is a duplication of this title in line 19. Which one do you want to keep?

File: docs/community/glossary.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
+# OpenSauced Glossary
+Welcome to the glossary, where we define concepts used on our platform to help you get the most out of OpenSauced.
+
+## PR Velocity
+The PR Velocity measures the percentage and amount of days that pull requests that are merged. It is a great way to track the progress of your open source contributions.
```
Comment (adiati98) [inline docs/community/glossary.md:5]: Let's add a space between the title and paragraph here.
Besides that it is the [best practice](https://www.markdownguide.org/basic-syntax/#heading-best-practices), we will do this going forward for consistency.

```suggestion
## PR Velocity

The PR Velocity measures the percentage and amount of days that pull requests that are merged. It is a great way to track the progress of your open source contributions.
```

**Review by adiati98 — CHANGES_REQUESTED** [review]
Hi @Lymah123,
I've left some feedback for you here. :)

**Comment by Lymah123** [comment]
> @Lymah123 you added a [setting-up-a-new-repository.md](docs/maintainers/setting-up-a-new-repository.md) here. I don't think that it is part of the newest Maintainer section.
> 
> @BekahHW can you confirm this?

Hi @adiati98 , I didn't change anything from that file.  I only touched it due to merge conflicts I had.

**Pull request closed without merging**